Source/Game/Game/Rendering/Terrain/TerrainLoader.cpp

```diff
@@ -28,7 +28,7 @@
 
 namespace fs = std::filesystem;
 
-AutoCVar_Int CVAR_TerrainLoaderPhysicsEnabled("terrainLoader.physics.enabled", "enable loading the terrain into the physics engine", 0, CVarFlags::EditCheckbox);
+AutoCVar_Int CVAR_TerrainLoaderPhysicsEnabled("terrainLoader.physics.enabled", "enable loading the terrain into the physics engine", 1, CVarFlags::EditCheckbox);
```

Comment:
Prefer to have this disabled until we can ensure higher performance, but I'll assume for now that this is simply a local change accidentally uploaded due to changes below. Please ensure alignment below is correct